Formula/maven@3.0.rb

```diff
@@ -0,0 +1,52 @@
+class MavenAT30 < Formula
+  desc "Java-based project management"
+  homepage "https://maven.apache.org/"
+  url "https://archive.apache.org/dist/maven/maven-3/3.0.5/binaries/apache-maven-3.0.5-bin.tar.gz"
```

Comment:
Unfortunately they've this version is no longer on that mirror. Apache suggests a number of other mirrors to use, but I wasn't sure what the procedure was for picking a non-Apache mirror. What would you recommend?